apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-extra-details/user-job/user-job-card/user-job-card.component.html

```diff
@@ -1,52 +1,21 @@
 <nb-card class="user-job-card">
   <nb-card-body>
-    <p class="position">{{ job.position }}</p>
-    <div class="com-flex com-items-center com-ml-1">
-      <fa-icon [icon]="faBuilding" class="company"></fa-icon>
-      <p>
-        {{ job.company }}
-      </p>
-    </div>
-    <div class="com-flex com-items-center com-gap-1.5">
-      <nb-icon icon="pin" status="primary"></nb-icon>
-      <p class="location">
-        {{ job.location_type === EJobLocationType.REMOTE ? 'Remote' : job.location }}
-      </p>
-    </div>
-    <nb-tag-list class="tags">
-      <nb-tag [text]="job.category | titlecase" appearance="outline" size="tiny" status="info"></nb-tag>
-      <nb-tag
-        [text]="job.job_type | capitalizeAndRemoveUnderscore"
-        appearance="outline"
-        size="tiny"
-        status="info"
-      ></nb-tag>
-      <nb-tag
-        appearance="outline"
-        size="tiny"
-        status="info"
-        text="Min. {{ job.experience }} year{{ job.experience > 1 ? 's' : '' }} experience"
-      ></nb-tag>
-    </nb-tag-list>
-    <!--    <nb-toggle (checkedChange)="toggleJobStatus.emit()" [checked]="job.status === jobStatus.OPEN">-->
-    <!--      {{ job.status | titlecase }}-->
-    <!--    </nb-toggle>-->
```

Comment:
why do we remove this?

Reply:
Because here we need to call the job-list-card component